manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
```

Comment:
Why change the location of the eyaml config? I believe it should be in /etc/eyaml and not /etc/puppet or whatever confdir is.